db/migrate/20200604181750_add_photos_locked_to_taxa.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhotosLockedToTaxa < ActiveRecord::Migration
+  def change
+    add_column :taxa, :photos_locked, :boolean, default: false
```

Comment:
With a default set, could this also be `null: false`?

Reply:
Yup. Might be wise. This is also going to be a rarely-used column, though, and in most situations, a `null` value will evaluate as `false` anyway.